core/src/test/java/uk/gov/gchq/magmacore/service/SignPatternTestData.java

```diff
@@ -102,14 +103,19 @@ static void createSignPattern(final MagmaCoreDatabase db) {
         // Create KindOfPerson
         kindOfPersonIri = new IRI(TEST_BASE, "kindOfPerson1");
 
+        // Create a classOfPerson.
+        classOfPersonIri = new IRI(TEST_BASE, "class1OfPerson");
```

Comment:
`class10fPerson`?

Reply:
This is just a name for a `classOfPerson`, so it means 'class 1 of person' - it could easily be 'my colleagues' instead but I didn't want to try to invent a meaningful scenario.